src/js/actions/tools.js

```diff
@@ -243,7 +243,7 @@ define(function (require, exports) {
 
         // if we are in vector mask mode we should not change pointer policies 
         if (toolStore.getVectorMode()) {
-            return Promise.resolve();
+            return this.transfer(policy.syncAllPolicies);
```

Comment:
Policies are only synced at the end points of this function. Note above that the `removePointerPolicies` call does not sync the policies.